GVFS/GVFS.FunctionalTests/Tests/EnlistmentPerTestCase/RepairTests.cs

```diff
@@ -13,6 +13,16 @@ namespace GVFS.FunctionalTests.Tests.EnlistmentPerTestCase
     [Category(Categories.MacTODO.M4)]
     public class RepairTests : TestsWithEnlistmentPerTestCase
     {
+        [TestCase]
+        public void NoFixesNeeded()
+        {
+            this.Enlistment.UnmountGVFS();
+
+            this.Enlistment.Repair(false);

```

Comment:
Style nit: whenever you're passing in a literal, use a named parameter so we know what it means. E.g. `this.Enlistment.Repair(confirm: false)`